src/Carousel.tsx

```diff
@@ -43,11 +43,17 @@ export function Carousel({
 		scrollSliderPrevious(slider);
 	}
 
+	function uuid() {
+		return (
+			Math.random().toString(36).substring(2, 15) + Math.random().toString(36).substring(2, 15)
```

Comment:
Have you considered using [crypto.randomUUID](https://developer.mozilla.org/en-US/docs/Web/API/Crypto/randomUUID)? Does it has any downside for this purpose?

Reply:
Of course, my first option was that, but maybe in local develop crypto web api will not work because only runs in secure contexts (https)